Source/Scene/Scene.js

```diff
@@ -166,6 +170,14 @@ define([
         };
     };
 
+    function AsyncLoader(ray, primitives) {
```

Comment:
This and `_asyncLoaders` are named really generically for the context of something as broad as `Scene`.  Specifically what is loaded?